tonic-web/tonic-web/src/call.rs

```diff
@@ -0,0 +1,294 @@
+use std::error::Error;
+use std::pin::Pin;
+use std::task::{Context, Poll};
+
+use bytes::{Buf, BufMut, Bytes, BytesMut};
+use futures_core::{ready, Stream};
+use http::{header, HeaderMap, HeaderValue};
+use http_body::{Body, SizeHint};
+use tonic::Status;
+
+use self::content_types::*;
+
+pub(crate) mod content_types {
```

Comment:
Can you publish the implementations in this file somewhere so that they are accessible from other crates? I'm implementing a wasm grpc-web client and I think this file is completely reusable on the client side. I've copied it for now but would love to be able to just `use` it.

Reply:
Sure,  I can move them to a separate crate if they are useful elsewhere